test/with_dummyserver/test_proxy_poolmanager.py

```diff
@@ -77,6 +78,10 @@ def teardown_class(cls) -> None:
         super().teardown_class()
         shutil.rmtree(cls.certs_dir)
 
+    @pytest.mark.xfail(
```

Comment:
There's a lot of duplication, can we create a marker and reuse it?

Reply:
Sure